File: Samples/Layout/HelloWorld/C#/HelloWorld.cs
using BitMiracle.Docotic.Pdf.Layout;

namespace BitMiracle.Docotic.Pdf.Samples
{
    class HelloWorld
    {
        public static void CreatePdf()
        {
            PdfDocumentBuilder.Create().Generate("hello.pdf", doc => doc.Pages(page =>
            {
                page.Content().Text("Hello, world!");
            }));
        }
    }
}

File: Samples/Layout/HelloWorld/VB.NET/HelloWorld.vb
Imports BitMiracle.Docotic.Pdf.Layout

Namespace BitMiracle.Docotic.Pdf.Samples
    Class HelloWorld
        Public Shared Sub CreatePdf()
            PdfDocumentBuilder.Create().Generate("hello.pdf",
                Sub(doc)
                    doc.Pages(Sub(page) page.Content().Text("Hello, world!"))
                End Sub)
        End Sub
    End Class
End Namespace
